src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.nio.file.Path;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static final String RESOURCES_PATH = "src/test/resources/";
+    private static FileReaderService reader;
+
+    @Test(expected = RuntimeException.class)
+    public void read_withNullPath_notOk() {
```

Comment:
i think we should rename this.
`<methodUnderTest>_<state>_<expectedBehavior>`
you are testing readFile(), not read()

Reply:
+Corrected.